<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Window</string>
  </property>
  <widget class="QTextEdit" name="text_info">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>441</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="list_script">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>311</width>
     <height>351</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_export">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text_error">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>441</width>
     <height>161</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>2</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_open_source">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="source_path">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>651</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Target</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="target_path">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>651</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_open_target">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_select_all">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_select_none">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select None</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Avaliable</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_select_type">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Script</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Instrument</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
